### DISCUSSION TITLE
Just try to install the 3.0.9 quickstart package — Errors and no installation?

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#554] [Q&A]
I just try to install the 3.0.9 quickstart package — I get Errors and nothing is installed?
**_— I use the normal Joomla Extensions Install_** 

**Errors:** 
JInstaller: :Install: Can't find XML setup file.
The Joomla package cannot be installed through the Extension Manager. Please use the Joomla! Update component to update Joomla.
JInstaller: :Install: Can't find XML setup file.
The Joomla package cannot be installed through the Extension Manager. Please use the Joomla! Update component to update Joomla.

**Comment by mac-joomla** [comment]
Hello @Chacapamac,

the quickstart package is a Joomla installer to install a fresh Joomla with astroid template already on board. 
It is not for installing from out of an existing Joomla installation.

**Comment by Chacapamac** [comment]
I see, my mistake, Thanks You!
